diff --git a/src/winetricks b/src/winetricks
index 6597cc8..1ef165d 100755
--- a/src/winetricks
+++ b/src/winetricks
@@ -5476,6 +5476,24 @@ load_gfw()
 
 #----------------------------------------------------------------
 
+w_metadata glu32 dlls \
+    title="" \
+    publisher="Microsoft" \
+    year="" \
+    media="download" \
+    file1="../xpsp3jp/WindowsXP-KB936929-SP3-x86-JPN.exe" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/glu32.dll"
+
+load_glu32()
+{
+    helper_xpsp3jp i386/glu32.dl_
+
+    w_try_cabextract -d "$W_SYSTEM32_DLLS" -L "$W_TMP"/i386/glu32.dl_
+    w_override_dlls native,builtin glu32
+}
+
+#----------------------------------------------------------------
+
 w_metadata glut dlls \
     title="The glut utility library for OpenGL" \
     publisher="Mark J. Kilgard" \
